packages/itwinui-react/src/core/ExpandableBlock/ExpandableBlock.tsx

```diff
@@ -129,8 +141,9 @@ export const ExpandableBlock = React.forwardRef((props, ref) => {
       <Box
         aria-expanded={expanded}
         className='iui-header'
+        aria-disabled={disabled}
```

Comment:
`aria-disabled` (and `aria-expanded`) is [not supported](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-disabled#associated_roles) on a generic role-less div.

Suggestion: add `role='button'` to this element (for now) and then test that it works in [NVDA](https://webaim.org/articles/nvda/).

In a future PR, we can improve this component to use `<button>` instead of `<div role='button'>`. Further reading: https://adrianroselli.com/2020/05/disclosure-widgets.html

Reply:
added role='button', seems to register as a button on NVDA

Reply:
https://github.com/iTwin/iTwinUI/assets/44583237/4fbf04b7-ec38-40b7-9104-732136fd2bd6